check-deps/Makefile

```diff
@@ -63,6 +63,6 @@ $(CHKDIR)/check-tstamp: PHONY
 	@if [ `id -u` -eq 0 ]; then chmod 666 $@; fi
 
 check-clean:
-	@$(RM) $(shell find $(CHKDIR) -type f -executable 2> /dev/null) $(CHKDIR)/check-tstamp $(CHKDIR)/*.o
+	@$(RM) $(shell find $(CHKDIR) -type f -perm -u=x,g=x,o=x 2> /dev/null) $(CHKDIR)/check-tstamp $(CHKDIR)/*.o
```

Comment:
Hmm.. so that means the overlayfs sets the 'x' bit somewhere for entires files?

Actually I'm ok with the change but I think `-perm 0755` would have the same effect.

Reply:
> Hmm.. so that means the overlayfs sets the 'x' bit somewhere for entires files?

Yes, it's matched for entire files so they are all removed.

> I think -perm 0755 would have the same effect.

Actually, it doesn't work in my environment.
```
$ ls -lrt check-deps/ | grep "x "
-rwxrwxr-x 1 honggyu honggyu 8464 Apr 26 19:27 clock_without_librt
-rwxrwxr-x 1 honggyu honggyu 8216 Apr 26 19:27 cc_has_mfentry
-rwxrwxr-x 1 honggyu honggyu 8456 Apr 26 19:27 cxa_demangle
-rwxrwxr-x 1 honggyu honggyu 8520 Apr 26 19:27 have_libelf
-rwxrwxr-x 1 honggyu honggyu 8216 Apr 26 19:27 cc_has_mno_sse2
-rwxrwxr-x 1 honggyu honggyu 8448 Apr 26 19:27 have_libpython2.7
-rwxrwxr-x 1 honggyu honggyu 8448 Apr 26 19:27 have_libpython3
-rwxrwxr-x 1 honggyu honggyu 8496 Apr 26 19:27 have_libluajit
-rwxrwxr-x 1 honggyu honggyu 8512 Apr 26 19:28 perf_clockid
-rwxrwxr-x 1 honggyu honggyu 8520 Apr 26 19:28 perf_context_switch
-rwxrwxr-x 1 honggyu honggyu 8528 Apr 26 19:28 have_libncurses
-rwxrwxr-x 1 honggyu honggyu 8512 Apr 26 19:28 have_libdw
-rwxrwxr-x 1 honggyu honggyu 8568 Apr 26 19:28 have_libunwind
-rwxrwxr-x 1 honggyu honggyu 8512 Apr 26 19:28 have_libcapstone
-rwxrwxr-x 1 honggyu honggyu 8232 Apr 26 19:28 cc_has_minline_all_stringops
```
It has to be 0775 in this case. But I think we better explicitly check only for executable permission as I currently uploaded.

Reply:
Ok, then could you please leave a comment on the line?  I think it's likely that I'll be confused by this later :)

Reply:
Sure. I've just updated it with a comment.

Reply:
Just curious, what's the permission of other files?